PKPD/inference/inference.py

```diff
@@ -106,8 +106,12 @@ def set_parameter_boundaries(self, boundaries:List):
         Arguments:
             boundaries {List} -- List of two lists. [min values, max values]
         """
-        min_values, max_values = boundaries[0], boundaries[1]
-        self.parameter_boundaries = pints.RectangularBoundaries(min_values, max_values)
+        print('boundaries are ' + str(boundaries))
```

Comment:
remove changes.

This function is only executed if boundaries is not None.

If you, nevertheless, would like to check the type/format of `boundaries`, you should do that with assertions, such that the function call breaks if a non-valid argument is passed ( we can talk about it if you like).

Reply:
there are two possible executions of the function and one of them passes None (if the boundary conditions are turned off) - left in as discussed